src/tests/tests.rs

```diff
@@ -0,0 +1,14 @@
+use super::Histogram;
+
+#[path = "helpers.rs"]
```

Comment:
Interesting... I view the multiple commits that are a part of a single PR that way -- as the thing I would bisect further if the merge of that PR introduced a bug. But in #18 vs #16, if #18 introduced a bug, then clearly #16 was not at fault. And furthermore, #16 *could* be modified pretty easily to avoid the change in #18. The two changes are basically orthogonal.

I'm happy to not be as strict about this though. This is slowly but surely becoming as much your project as mine, and if that's how you prefer your merges, I'm okay with that :+1: